src/app/system/manage-data-tables/view-data-table/view-data-table.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+
+    <button mat-raised-button color="primary" [routerLink]="['edit']">
+        <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+        Edit
+    </button>
+
+    <button mat-raised-button color="warn" (click)="delete()">
+        <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+        Delete
+    </button>
+
+</div>
+
+<div class="container m-b-15" fxLayoutAlign="start">
```

Comment:
```suggestion
<div class="container m-b-15">
```
Add another class for ensuring a proper width instead of container as it covers the entire row.

Reply:
It works well, the only problem I can see that the card and the buttons don't come in a single row.